src/main/java/edu/ucsb/cs156/example/entities/PersonalSchedule.java

```diff
@@ -0,0 +1,35 @@
+package edu.ucsb.cs156.example.entities;
+
+import javax.persistence.Entity;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.GeneratedValue;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Entity(name = "personalschedule")
+public class PersonalSchedule {
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private long id;
+
+  // This establishes that many todos can belong to one user
+  // Only the user_id is stored in the table, and through it we
+  // can access the user's details
+
+  @ManyToOne
+  @JoinColumn(name = "user_id")
+  private User user;
+  private String Name;
```

Comment:
Variables must be lowercase (camel case is standard for Java).  This is important because I already worked ahead in the PersonalSchedule Fixtures and for PersonalScheduleForm component where I had to use these variables (but in camel case for as standard in Java).

Reply:
Agreed... this needs to be changed